src/host/buildvm_asm.c

```diff
@@ -387,7 +387,7 @@ void emit_asm(BuildCtx *ctx)
 #if !(LJ_TARGET_PS3 || LJ_TARGET_PSVITA)
     fprintf(ctx->fp, "\t.section .note.GNU-stack,\"\"," ELFASM_PX "progbits\n");
 #endif
-#if LJ_TARGET_PPC && !LJ_TARGET_PS3 && !LJ_ABI_SOFTFP
+#if LJ_TARGET_PPC && !LJ_TARGET_PS3 && !LJ_ABI_SOFTFP && !__clang__
```

Comment:
That doesn't look correct.  It avoids emitting the .gnu_attribute for clang.  Is the attribute not necessary at all for powerpc hard float builds, i.e. does the compiler end up doing the right thing regardless of this attribute?

Reply:
After excluding the attribute ".gnu_attribute" the build ran successfully. I shall withdraw this PR for now to investigate more on the float builds and will update accordingly.

Reply:
Sounds good, thanks!